tutorials/tracing/openai_tracing_tutorial.ipynb

```diff
@@ -0,0 +1,398 @@
+    "\n",
+    "While it's possible to produce structured output without using function calling via careful prompting, function calling is more reliable at producing output that conforms to a particular format. For more details on OpenAI's function calling API, see the [OpenAI documentation](https://platform.openai.com/docs).\n",
+    "\n",
+    "Let's get started!\n",
+    "\n",
+    "ℹ️ This notebook requires an OpenAI API key."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Install Dependencies and Import Libraries\n",
+    "\n",
+    "Install dependencies."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install arize-phoenix jsonschema openai"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Import libraries."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import os\n",
+    "from getpass import getpass\n",
+    "from typing import Any, Dict, Literal, TypedDict\n",
+    "\n",
+    "import jsonschema\n",
+    "import openai\n",
+    "import pandas as pd\n",
+    "import phoenix as px\n",
+    "from phoenix.trace.exporter import HttpExporter\n",
+    "from phoenix.trace.openai import OpenAIInstrumentor\n",
+    "from phoenix.trace.tracer import Tracer\n",
+    "from tenacity import (\n",
+    "    retry,\n",
+    "    stop_after_attempt,\n",
+    "    wait_random_exponential,\n",
+    ")\n",
+    "\n",
+    "pd.set_option(\"display.max_colwidth\", None)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Configure Your OpenAI API Key\n",
+    "\n",
+    "Set your OpenAI API key if it is not already set as an environment variable."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if not (openai_api_key := os.getenv(\"OPENAI_API_KEY\")):\n",
+    "    openai_api_key = getpass(\"🔑 Enter your OpenAI API key: \")\n",
+    "openai.api_key = openai_api_key\n",
+    "os.environ[\"OPENAI_API_KEY\"] = openai_api_key"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Instrument Your OpenAI Client\n",
+    "\n",
+    "Instrument your OpenAI client with a tracer that emits telemetry data in OpenInference format. [OpenInference](https://arize-ai.github.io/open-inference-spec/trace/) is an open standard for capturing and storing LLM application traces that enables LLM applications to seamlessly integrate with LLM observability solutions such as Phoenix."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tracer = Tracer(exporter=HttpExporter())\n",
```

Comment:
Might just nix this to keep it simple

Reply:
By default, our tracer does not have an exporter. We would need to change the default to `HttpExporter` in order for traces to show up in Phoenix. https://github.com/Arize-ai/phoenix/blob/5128abe0add224c506aba7cce3b06389818eb09c/src/phoenix/trace/tracer.py#L57